{"name": "Проверка наличия ошибки при неправильном логине или пароле", "status": "passed", "parameters": [{"name": "login", "value": "'falselogin'"}, {"name": "password", "value": "'rightpassword'"}, {"name": "expected_status_code", "value": "404"}, {"name": "response_text", "value": "'{\"code\":404,\"message\":\"Учетная запись не найдена\"}'"}], "start": 1730934778387, "stop": 1730934778467, "uuid": "cd015974-8155-468b-a9eb-ffa16f7164f4", "historyId": "7ad058b20741e3b70aaa252167c8c6b6", "testCaseId": "95b67cc52f34af66d939aef0fd27906f", "fullName": "tests.courier.test_login_courier.TestLoginCourier#test_login_courier_false_fields", "labels": [{"name": "feature", "value": "Логин курьера"}, {"name": "parentSuite", "value": "tests.courier"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "mariamna-pc-kb2404"}, {"name": "thread", "value": "20070-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.courier.test_login_courier"}]}